Black-Market/Black-Market/Models/UserDataResponse.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UserResponse.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+
+struct UserDataResponse: Codable {
+    let accessToken: String
+    let refreshToken: String
+    let user: UserModel
+    
+    
+    enum CodingKeys: String, CodingKey {
+        case accessToken = "access_token"
+        case refreshToken = "refresh_token"
```

Comment:
you can use the `keyDecodingStrategy = .convertFromSnakeCase` for the JSONDecoder if this is always or mostly be the case

Reply:
Done